packages/app/src/__tests__/NexusPack.test.tsx

```diff
@@ -89,6 +89,24 @@ test('visualize NXdata group with "rgb-image" interpretation', async () => {
   expect(tabs[0]).toHaveTextContent(NexusVis.NxRGB);
 });
 
+test('visualize NXdata group with unknown interpretation', async () => {
```

Comment:
We should add somewhere (comment or test title) that the NXdata group should also have more than 1 dimension. Otherwise, the vis will be `NxSpectrum` and not `NxImage`.